scss/components/badge.scss

```diff
@@ -16,6 +16,7 @@ $block: ns(badge);
     border-style: solid;
     padding: fd-space(1) fd-space(2);
     vertical-align: middle;
+    margin-left: fd-space(reg);
```

Comment:
This is too much space and we can't insure that the first item should always have a left margin. What if I create a new `div` to hold my badges then I'll have an extra margin.

Make these `inline-block` instead so they mimic a text flow. Also, change the padding to force these to be 24px tall instead of 26px. They should be the same height as the line-height of `p` elements.

```
display: inline-block;
padding: fd-space(1)-1px fd-space(2);
```

And remove the  `line-height: fd-space(reg);` rule. It's not needed with the adjustments above.

Reply:
Make the same adjustment to the labels if necessary.